### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by davemcorwin**
all player and game tracking features implemented. wins and losses are recorded and reflected in leaderboard.

utilizing meteor methods to handle database changes and assuming one way data flow through app, with one commented exception.

this is all ripe for eventual refactoring.

### CODE REVIEW

**Comment by davemcorwin** [comment]
make sure to `meteor reset` to get the database squared away

File: client/components/choose-player-menu.jsx [hunk 1]
```diff
@@ -39,8 +37,8 @@ ChoosePlayerMenu = React.createClass({
 
   render() {
 
-    var menuItems = this.data.players.map((player) => {
-      return { title: player.name };
+    let menuItems = this.data.players.map((player) => {
```
Comment (bhennes2) [inline client/components/choose-player-menu.jsx:40]: @dshvazz Are you just experimenting with `let` here or is there a scoping reason to do it?

File: client/components/leaders.jsx [hunk 2]
```diff
@@ -1,31 +1,112 @@
 Leaders = React.createClass({
 
-  mixins: [ReactRouter.Navigation],
+  mixins: [ReactRouter.Navigation, ReactMeteorData],
```
Comment (bhennes2) [inline client/components/leaders.jsx:3]: @dshvazz Cool. I'll probably need a lot of this sorting/leaderboard functionality for the admin stuff so I'll probably do a bit of abstraction here.

**Comment by bhennes2** [comment]
:+1:

**Pull request merged**